on_boot.d/80-udm-backup-ftp.sh

```diff
@@ -35,26 +31,31 @@ SDN_MOUNT="/mnt/data/unifi-os/unifi/data/backup/autobackup:/backups/unifi:ro"
 # you can comment next line to disable protect backup (or if protect is disabled on your UDM)
 PROTECT_MOUNT="/mnt/data_ext/unifi-os/unifi-protect/backups:/backups/protect:ro"
 
-CRON_CMD="${CRON_SCHEDULE} podman run -it --rm --name UDM-FTP-Backup --network=host -e \"FTP_SERVER=$FTP_SERVER\" -e \"FTP_PATH=$FTP_PATH\" -e \"FTP_USER=$FTP_USER\" -e \"FTP_PASSWORD=$FTP_PASSWORD\""
+CRON_CMD="${CRON_SCHEDULE} podman run -it --rm --name UDM-FTP-Backup --network=host --env-file='$ENV_FILE'"
 BACKUP_IMG='docker.io/aessing/udm-backup-ftp'
 
+if [ ! -r "$ENV_FILE" ]; then
+    echo "ERROR: File '$ENV_FILE' not found or is not readable!" 1>&2
+    echo "Please create it based on 'conf.env.dist' with your own values" 1>&2
+    exit 1
+fi
 
 
-if [ ! -z "${SDN_MOUNT}" ]; then
+if [ -n "${SDN_MOUNT:-}" ]; then
```

Comment:
Yes sure I can.

The `! -z` its to check if the variable is set, but this is done also with `-n` option ([the link to the doc](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html), it is almost at the end of the html).

The `${variable:-default value}`, set a default variable if variable is empty by doing `${SDN_MOUNT:-}` we are setting a default empty string which will return false to `! -z` or "more correct" `-n` even in a strict environment if at any point you use `set +e` setopt.

Here you can use `-n $SDN_MOUNT` if you want because that option is not set but just that is the way I used to write because I like to use `set -euo pipefail` to write a little bit more strict bash scripts that handle all possible errors (see my [.Sloth](https://github.com/gtrabanco/.Sloth) project if you want to; sorry for the Spam).